servicebox-app/src/main/resources/templates/credentials/list.html

```diff
@@ -2,9 +2,8 @@
 {#include base}
 {#title}Credential{/title}
 {#body}
-<div class="container">
-    <div class="btn btn-primary"><a href="/credentials/new" class="text-white">New credential</a></div>
-</div>
+<div class="container pt-4">
+<a href="/credentials/new" class="btn btn-info active" role="button" aria-pressed="true">New credential</a>
```

Comment:
If you add a div element here, then you need to align the rest of the elements within this new div.

Reply:
Fixed with 0578973